dist/OBS-statistics.R

```diff
@@ -0,0 +1,291 @@
+# We use ggplot2, a library that makes painting graphics much easier.
+# You can check the types of graphs and examples code here: https://plot.ly/ggplot2
+
+# REQUIRE NEEDED LIBRARIES, in case that you don't have then installed, use: install.packages("ggplot2")
```

Comment:
@evanrolfe What do you propose as an alternative?

I'm fine with adding the code of the R file by now. If R scripts grows a lot in number and size, hopefully, we can move them to another project later.